examples/c/ex_call_center.c

```diff
@@ -95,84 +89,71 @@ main(void)
 		{ 0, 0, 0, 0, NULL, NULL }
 	};
 
-	/*
-	 * Create a clean test directory for this run of the test program if the
-	 * environment variable isn't already set (as is done by make check).
-	 */
-	if (getenv("WIREDTIGER_HOME") == NULL) {
-		home = "WT_HOME";
-		ret = system("rm -rf WT_HOME && mkdir WT_HOME");
-	} else
-		home = NULL;
-
-	if ((ret = wiredtiger_open(home, NULL, "create", &conn)) != 0) {
-		fprintf(stderr, "Error connecting to %s: %s\n",
-		    home == NULL ? "." : home, wiredtiger_strerror(ret));
-		return (EXIT_FAILURE);
-	}
-	/* Note: further error checking omitted for clarity. */
+	home = example_setup(argc, argv);
+
```

Comment:
very minor nit: since `home` is used in the next line, I'd remove the blank line.